pypuf/experiments/experiment/logistic_regression.py

```diff
@@ -0,0 +1,99 @@
+import time
+from numpy.random import RandomState
+from numpy import amin, amax, mean, array, append
+from pypuf.experiments.experiment.base import Experiment
+from pypuf.learner.regression.logistic_regression import LogisticRegression
+from pypuf.simulation.arbiter_based.ltfarray import LTFArray
+from pypuf import tools
+
+
+class ExperimentLogisticRegression(Experiment):
+    """
+        This class cloud be used to setup an experiment with logistic regression learner on a LTFArray puf simulation.
```

Comment:
"This Experiment uses the logistic regression learner on an LTFArray PUF simulation."